databricks/koalas/tests/test_dataframe.py

```diff
@@ -1200,13 +1201,15 @@ def test_transpose(self):
             columns=['score', 'kids', 'age'])
         kdf2 = ks.from_pandas(pdf2)
 
+        set_option("compute.max_rows", None)
         self.assertEqual(
```

Comment:
Let's do:

```python
try:
    self.assertEqual(
        ...
finally:
    reset_option("compute.max_rows")
```

to avoid this test failure affects other tests related to `compute.max_rows` option.